test/features/user_records_steps_test.rb

```diff
@@ -2,15 +2,17 @@
 
 class UserRecordsSteps < ActionDispatch::IntegrationTest
   def test_user_adds_a_step
-    app = Application.create(:company => "Basecamp", :status => "open")
+    user = create(:person)
+    page.set_rack_session(user_id: user.id)
+    app = user.applications.create(company: "Basecamp", status: "applied")
     visit application_path(app)
 
     within(".application") do
       click_link_or_button 'add_step'
     end
 
-    select('feedback', :from => 'step_kind')
-    fill_in 'step_note', :with => "They're not hiring. Bummer!"
+    select('feedback', from: 'step_kind')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.